reana/reana_dev/cluster.py

```diff
@@ -376,11 +376,25 @@ def cluster_unpause():
     run_command(cmd, "reana")
 
 
+@click.option(
+    "-m",
+    "--mount",
+    "mounts",
+    multiple=True,
+    help="Which local path directories are to be deleted? [local_path:cluster_node_path]",
+)
 @cluster_commands.command(name="cluster-delete")
-def cluster_delete():
-    """Delete REANA cluster."""
-    cmd = "kind delete cluster"
-    run_command(cmd, "reana")
+def cluster_delete(mounts):  # noqa: D301
+    """Delete REANA cluster.
+
+    \b
+    Example:
+       $ reana-dev cluster-delete -m /var/reana:/var/reana
+    """
+    cmds = ["rm -rf '{}'".format(mount.split(":")[0]) for mount in mounts]
```

Comment:
(1) In some cases the `/var/reana` is owned by another person, so it may be good to do:

```
rm -rf '{}/*'
```

in the above command.

(2) Also, the command may be dangerous if somebody uses `-m ~/mydata:/path` -- let's beware! I'd say let's add a check and if the value of `mount` is not `/var/reana`, let's refuse to do it... just to be safe...